src/workerd/api/global-scope.c++

```diff
@@ -471,16 +471,28 @@ kj::Promise<WorkerInterface::AlarmResult> ServiceWorkerGlobalScope::runAlarm(kj:
             !jsg::isDoNotLogException(desc) && context.isOutputGateBroken()) {
           LOG_NOSENTRY(ERROR, "output lock broke during alarm execution", actorId, e);
         } else if (context.isOutputGateBroken()) {
-          // We don't usually log these messages, but it's useful to know the real reason we failed
-          // to correctly investigate stuck alarms.
-          LOG_NOSENTRY(ERROR,
-              "output lock broke during alarm execution without an interesting error description",
-              actorId, e);
+          if (e.getType() == kj::Exception::Type::OVERLOADED) {
+            // This is a bit hackish, but we know for a fact that this specific exception detail is
```

Comment:
Can we maybe use exception details here? https://github.com/capnproto/capnproto/blob/14132442b125d0383285e36809e467c6b6a759aa/c%2B%2B/src/kj/exception.h#L185

Reply:
Commented the same thing but had forgotten to submit my review :sweat_smile: